ent-mysql/routes/routes.go

```diff
@@ -0,0 +1,71 @@
+package routes
+
+import (
+	"context"
+	"log"
+	"strconv"
+
+	"ent-mysql/database"
+	"github.com/gofiber/fiber/v2"
+)
+
+var ctx = context.Background()
+
+func GetBook(c *fiber.Ctx) error {
+	id, _ := strconv.Atoi(c.Params("id"))
+	b, err := database.DBConn.Book.
+		Get(ctx, id)
+	if err != nil {
```

Comment:
We should check not found error here and return 404

Reply:
> We should check not found error here and return 404

Oh, I see. I just added the error handling ! :)